packages/react-native/ReactCommon/react/renderer/components/scrollview/ScrollViewProps.h

```diff
@@ -68,6 +68,7 @@ class ScrollViewProps final : public ViewProps {
   ContentInsetAdjustmentBehavior contentInsetAdjustmentBehavior{
       ContentInsetAdjustmentBehavior::Never};
   bool scrollToOverflowEnabled{false};
+  bool inverted{false};
```

Comment:
This isn't *quite* sufficient. You'll also need to update `ScrollViewProps.cpp`. It might also be worth giving an inverted list behaviors a whirl in Fabric.

Reply:
LGTM